src/DependencyInjection/Configuration.php

```diff
@@ -214,4 +215,23 @@ private function createSmsConfiguration(ArrayNodeDefinition $root)
                 ->end()
             ->end();
     }
+
+    private function createLocaleCookieConfiguration(ArrayNodeDefinition $root)
+    {
+        $root
+            ->children()
+                ->arrayNode('locale_cookie')
+                    ->info('Cookie settings for locale cookie')
+                    ->isRequired()
+                    ->children()
+                        ->scalarNode('name')->end()
+                        ->scalarNode('domain')->end()
+                        ->integerNode('expire')->end()
```

Comment:
I would like that (and maybe copy the nice cookie name preg_match), but in the end it's up to you as the project lead of course :stuck_out_tongue_winking_eye: